src/Teams/Models/Team.cs

```diff
@@ -7,15 +7,11 @@ namespace Teams.Models
     public class Team
     {
         [Key]
-        [Column ("Id", TypeName = "int")]
-        [Required(AllowEmptyStrings = false)]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column ("Id", TypeName = "int"), Required, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
-        [Column("TeamName", TypeName = "nvarchar(MAX")]
-        [Required(AllowEmptyStrings = true)]
+        [Column("TeamName", TypeName = "nvarchar(MAX"), Required]
         public string TeamName { get; set; }
-        [Column("TeamOwner", TypeName = "nvarchar(MAX")]
-        [Required(AllowEmptyStrings = true)]
+        [Column("TeamOwner", TypeName = "nvarchar(MAX"), Required(AllowEmptyStrings = false)]
```

Comment:
The same issue that were discussed 1000 times. The type constraint is invalid and this is a foreign key and you can skip types for this type of column, EF will set the correct type automatically if the FK is defined correctly in the model